app/src/main/java/org/fossasia/openevent/app/event/attendees/AttendeesPresenter.java

```diff
@@ -41,8 +43,14 @@ public void attach(long eventId, IAttendeesView attendeesView) {
     }
 
     @Override
-    public void start() {
-        loadAttendees(false);
+    public void start(boolean forceStart) {
+        if (forceStart)
+            progress = 0;
+        if (progress == 0) {
+            loadAttendees(false);
+        } else {
+            attendeesView.showAttendees(attendeeList);
+        }
```

Comment:
>In view, we could check that if it is rotated and loading again

view doesn't have any idea if data is loaded or not. Presenter has. Thats why I am checking it in presenter.